src/DependencyInjection/DI/src/ServiceLookup/ILEmit/ILEmitResolverBuilder.cs

```diff
@@ -16,8 +17,6 @@ internal sealed class ILEmitResolverBuilder : CallSiteVisitor<ILEmitResolverBuil
         private static readonly MethodInfo ResolvedServicesGetter = typeof(ServiceProviderEngineScope).GetProperty(
```

Comment:
Not part of this change, but it is a little weird to have TResult be Expression since it's never used. Probably makes sense to change it to object like TArgument in ExpressionResolverBuilder.